src/main/resources/templates/db_scripts/voucher_number_sequence.sql

```diff
@@ -1,2 +1,3 @@
 CREATE SEQUENCE IF NOT EXISTS ${myuniversity}_${mymodule}.voucher_number MINVALUE 0 NO MAXVALUE CACHE 1 NO CYCLE;
 GRANT USAGE ON SEQUENCE ${myuniversity}_${mymodule}.voucher_number TO ${myuniversity}_${mymodule};
```

Comment:
Is it still required taking into account that following line makes tenant user an owner of this sequence?